### PR TITLE
Use a tip as block id instead of creating a new block

### LINKED ISSUE #64: Unify "Get a Block" example
## Description

The [Get a Block](https://wiki.iota.org/shimmer/iota.rs/how_tos/get_block/) examples explain different features in different languages.
This issue is part of the efforts in iotaledger/iota-sdk#59 

## Motivation

We should show/explain the same features in all languages (if they exist).

## Requirements

Let's show in every example how to:

- [ ] Don't show again on how to create a block. We already know that. Instead do it the Rust way and use a random tip. (That way we even include new info on how to fetch tips :trollface: ) We could probably even get rid of the get id from env part in the rust example
    - [ ] Rust (Remove env part and just use a random tip)
    - [ ] Nodejs
    - [ ] Python
    - [ ] Java

- [ ] Explain how to get block metadata
    - [ ] Rust
    - [x] Nodejs
    - [x] Python
    - [ ] Java 

## Are you planning to do it yourself in a pull request?

No.

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Use a tip as block id instead of creating a new block

## Links to any relevant issues

Fixes #64

## Type of change

- Documentation Fix

## How the change has been tested

Running the examples

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Dr-Electron** [comment]
🏃‍♂️

**Comment by Dr-Electron** [comment]
Didn't test the code, but looks good to me

**Comment by DaughterOfMars** [comment]
For those of us simpletons who are less familiar with the systems, could you provide an explanation for why this is okay to do?

**Comment by Thoralf-M** [comment]
@Alexandcoats there must always be at least one tip (block that isn't referenced by other blocks yet) at the end of the Tangle
Therefore we can always retrieve a new block, which we can then also request from the node without fearing that it has already been pruned

**Review by DaughterOfMars — APPROVED** [review]

File: sdk/examples/client/get_block.rs
Comment (thibault-martinez) [inline sdk/examples/client/get_block.rs:1]: I don't have anything against that example but we already had get_block and get_block_metadata with tip, right?

Comment (thibault-martinez) [reply sdk/examples/client/get_block.rs:1]: Ah alright you wanted one that does both for consistency, ignore me

**Review by thibault-martinez — APPROVED** [review]

File: bindings/nodejs/examples/client/07_get_block_data.ts [hunk 1]
```diff
@@ -20,16 +20,16 @@ async function run() {
...
-        const blockIdAndBlock = await client.buildAndPostBlock();
-        console.log('Block:', blockIdAndBlock, '\n');
+        // Fetch a block ID from the node.
+        const blockIds = await client.getTips();
+        console.log('Block IDs:', blockIds, '\n');
 
         // Get the metadata for the block.
-        const blockMetadata = await client.getBlockMetadata(blockIdAndBlock[0]);
+        const blockMetadata = await client.getBlockMetadata(blockIds[0]);
         console.log('Block metadata: ', blockMetadata, '\n');
```
Comment (Alex6323) [inline bindings/nodejs/examples/client/07_get_block_data.ts:29]: so ... we are only interested in ONE block id, right? Isn't it enough then to log only this one block id?

Comment (Thoralf-M) [reply bindings/nodejs/examples/client/07_get_block_data.ts:29]: Yes, might change it in another PR :+1:

**Pull request merged**